src/dashboard/src/settings/common.py

```diff
@@ -340,3 +360,15 @@ def get_env_variable(var_name):
 TEXTAREA_WITH_HELP_ATTRS = {'rows': '4', 'class': 'span11 has_contextual_help'}
 INPUT_ATTRS = {'class': 'span11'}
 INPUT_WITH_HELP_ATTRS = {'class': 'span11 has_contextual_help'}
+
+SHIBBOLETH_REMOTE_USER_HEADER = 'HTTP_EPPN'
```

Comment:
This is the header that comes from nginx and identifies the user. It will generally be an email address.